### ISSUE TITLE
[agw] SCTP connection drops out after multiple S1AP test runs due to "Too many open files"

### ISSUE THREAD

**Issue by ardzoht** [magma/magma#8898]
### Your Environment

- **Version:** master
- **Affected Component:** Access Gateway
- **Affected Subcomponent:** sctpd, MME
- **Deployment Environment:** Vagrant (AGW)

### Describe the Issue

- sctpd connection drops after running multiple s1ap tests (test that has been used to reproduce is `test_multi_enb_partial_reset.py`)
- After running hundreds of iterations (e.g. 300) there is an occurrence of the test getting stuck on S1 Setup step
- sctpd logs:
```
Aug 30 19:35:23 magma-dev-focal sctpd[24713]: I0830 19:35:23.061749 25486 sctp_connection.cpp:185] SCTP_SHUTDOWN_EVENT received
Aug 30 19:35:23 magma-dev-focal sctpd[24713]: I0830 19:35:23.061753 25486 sctp_connection.cpp:280] Sending close connection for assoc_id 1002
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.246320 25486 sctp_connection.cpp:167] HandleClientSock sd = 1020
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.246387 25486 sctp_connection.cpp:189] SCTP association change event received
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.247225 25486 sctp_connection.cpp:167] HandleClientSock sd = 1020
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.247238 25486 sctp_connection.cpp:193] Unhandled notification type 32776
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.247242 25486 sctp_connection.cpp:167] HandleClientSock sd = 1021
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.247246 25486 sctp_connection.cpp:189] SCTP association change event received
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.247778 25486 sctp_connection.cpp:167] HandleClientSock sd = 1020
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.247790 25486 sctp_connection.cpp:220] [sd:1020] msg of len 51 on 1003:0
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.248373 25486 sctp_connection.cpp:167] HandleClientSock sd = 1021
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.248389 25486 sctp_connection.cpp:193] Unhandled notification type 32776
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.248394 25486 sctp_connection.cpp:167] HandleClientSock sd = 1022
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.248397 25486 sctp_connection.cpp:189] SCTP association change event received
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: I0830 19:35:33.248754 25486 sctp_connection.cpp:131] accept error (24): Too many open files
Aug 30 19:35:33 magma-dev-focal sctpd[24713]: terminate called without an active exception
Aug 30 19:35:33 magma-dev-focal systemd[1]: sctpd.service: Main process exited, code=killed, status=6/ABRT
Aug 30 19:35:33 magma-dev-focal systemd[1]: sctpd.service: Failed with result 'signal'.
Aug 30 19:35:38 magma-dev-focal systemd[1]: sctpd.service: Scheduled restart job, restart counter is at 26.
Aug 30 19:35:46 magma-dev-focal systemd[1]: Stopped Magma sctpd service.
Aug 30 19:35:46 magma-dev-focal systemd[1]: Starting Magma sctpd service...
Aug 30 19:35:46 magma-dev-focal sctpd[170641]: /bin/cp: cannot stat '/usr/local/share/sctpd/version': No such file or directory
Aug 30 19:35:48 magma-dev-focal ovs-vsctl[170720]: ovs|00001|vsctl|INFO|Called as ovs-vsctl --all destroy Flow_Sample_Collector_Set
Aug 30 19:35:48 magma-dev-focal ovs-vsctl[170789]: ovs|00001|vsctl|INFO|Called as ovs-vsctl --timeout=5 -- --if-exists del-port uplink_br0 dhcp0
```

**To Reproduce**
- Run `test_multi_enb_partial_reset.py` 300 times, wait for one iteration to get stuck on S1 Setup

**Expected behavior**
- S1 connection should not be dropping, S1AP tests are sequential and should be idempotent.

**Comment by ssanadhya** [comment]
@VinashakAnkitAman , could you please summarize here what you found on this issue so far? I can also spend some time on this next week.

**Comment by sebathomas** [comment]
Should be fixed by #12931.
